src/PageContainer/component.js

```diff
@@ -100,8 +100,10 @@ export default class PageContainer extends Component {
     }
 
     const item = find(context.collection, pageUrl)
-
+    console.log(isClient)
+    console.log(typeof window !== "undefined")
```

Comment:
Got it. So change isClient to a function then ?

anyway, I need to pull babel-plugin-rewire browserHistory. So this PR is blocked by #286